sql/mob_droplist.sql

```diff
@@ -15867,7 +15867,7 @@ INSERT INTO `mob_droplist` VALUES (2996,0,0,1000,15614,@UNCOMMON);
 INSERT INTO `mob_droplist` VALUES (2997,0,0,1000,2851,273);
 INSERT INTO `mob_droplist` VALUES (2998,0,0,1000,11339,340);
 INSERT INTO `mob_droplist` VALUES (2999,0,0,1000,11338,111);
-INSERT INTO `mob_droplist` VALUES (3000,0,0,1000,19305,111);
+INSERT INTO `mob_droplist` VALUES (3000,0,0,1000,19305,360);
```

Comment:
please comment what the item is.

also, that droprate isnt correct. I'll link a chart